spec/order_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'order'
+
+describe Order do
+  it "Expects order to be an instance of Order class" do
+    expect(subject).to be_an_instance_of(Order)
+  end
+
+  it "Expects order to respond to add_to_order" do
+    expect(subject).to respond_to(:add_to_order)
+  end
+
+  it "Expects order to respond to total_of_order" do
+    expect(subject).to respond_to(:order_total)
+  end
```

Comment:
I really like your use of these unit tests to show how the program has been built up. 

As a refactoring suggestion, I think it is possible to delete them after you have written further tests, as my understanding is that they are implicitly tested by the subsequent tests. i.e. line 9 should be implicitly tested by line 20